overlay/default.nix

```diff
@@ -124,6 +124,36 @@ in
         cd ${src}
         ${final.cabal-install}/bin/cabal check
       '';
+
+      weeder = { weederToml, hs-pkgs }: let
```

Comment:
I propose to leave a comment that `writeHieFiles = true` is mandatory for packages that are supposed to be checked by weeder